RoboticonColony/Assets/Code/RoboticonCustomisation.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+
+sealed public class RoboticonCustomisation
+{
+    public string Name { get; private set; }
+
+    /// <summary>
+    /// Prerequisite customisations that a roboticon must have in order for this one to be applied
+    /// </summary>
+    public List<RoboticonCustomisation> Prerequisites { get; private set; }
```

Comment:
Could this not be simplified from a list to just a single RoboticonCustomisation instance? Any one customisation only requires 1 prerequisite, power I requires base, power II requires power I etc